src/software/sensor_fusion/sensor_fusion.h

```diff
@@ -87,7 +102,7 @@ class SensorFusion : public Subject<World>,
      *
      * @return enemy team from vision_detection
      */
-    Team getEnemyTeamFromvisionDetecion(const VisionDetection &vision_detection);
+    Team getEnemyTeamFromVisionDetection(const VisionDetection &vision_detection);
 
     // Objects used to aggregate and store state. We use these to aggregate the state
     // so that we always publish "complete" data, not just data from a single frame/
```

Comment:
ALso may be out of scope, but it seems we don't need the `state` variables. They aren't used to update the world, and aren't updated at all (despite sometimes being used in _other_ update functions, which is bad since their state if out of date). I think we can remove them pretty easily and it will be less to manage.